client/scripts/controllers/server/chain_events.ts

```diff
@@ -8,7 +8,13 @@ class ChainEventsController {
   }
 
   public offences(callback) {
```

Comment:
This callback style is inconsistent with the rest of the code, which uses promises.